base/system/services/services.rc

```diff
@@ -51,3 +51,6 @@
 #ifdef LANGUAGE_ZH_TW
     #include "lang/zh-TW.rc"
 #endif
+#ifdef LANGUAGE_HE_IL
```

Comment:
Please move this block in lexicographical order, between the language names that precede and follow it.